src/jpegParser.js

```diff
@@ -228,8 +241,13 @@
     var processMetaData = function(error, metaData) {
       var thumbnailMetaData = metaData && metaData.thumbnailMetaData;
       var thumbnailBlob = metaData && metaData.thumbnailBlob;
+      var imageSize = metaData && {
+        width: metaData.width,
+        height: metaData.height,
+        progressive: metaData.progressive
+      };
       metaData = metaData && metaData.Exif;
-      callback(error, metaData, thumbnailMetaData, thumbnailBlob);
+      callback(error, metaData, thumbnailMetaData, thumbnailBlob, imageSize);
```

Comment:
Sure. But I didn't want to break the API. If I do that I have to fix the documentation, the demo, the tests.